app/bundles/PageBundle/Controller/PublicController.php

```diff
@@ -402,6 +404,10 @@ public function trackingAction()
         /** @var TrackingHelper $trackingHelper */
         $trackingHelper = $this->get('mautic.page.helper.tracking');
         $sessionValue   = $trackingHelper->getSession(true);
+        
+        /** @var \Mautic\PageBundle\Model\PageModel $model */
+        $model = $this->getModel('page');
+        $model->hitPage(null, $this->request, '200', $lead);
```

Comment:
Passing `$lead` completely bypasses the logic in [PageModel](https://github.com/mautic/mautic/blob/staging/app/bundles/PageBundle/Model/PageModel.php#L462-L479). This alters current application behaviour. E.g. `ct` query parameter is not taken into account, etc.